scripts/bundleSize/bundleSizeConfig.js

```diff
@@ -5,5 +5,5 @@ module.exports = {
   // below the smallest value in the build output; this avoids the
   // need for frequent changes as bundle sizes fluctuate.
   MIN_SIZE: 574,
-  MAX_SIZE: 877,
+  MAX_SIZE: 891,
```

Comment:
That's a big jump, this consent banner must be a lot heavier :( Maybe worth a look in the bundle report to confirm

Reply:
In fact is this bringing new emotion from psammead along with it? Is that okay, I feel like @ryanmccombe said it was? _Probably_ just for this component though..

Reply:
99% sure it’s fine because the emotion 11 change was in 5.3.1 and the version in this branch's `package.json` is pinned to 5.3.0

if the branch passes amp validation, we're good